Ports/libwebp/package.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='libwebp'
+version='1.3.1'
+files=(
+    "https://github.com/webmproject/libwebp/archive/refs/tags/v${version}.tar.gz 1c45f135a20c629c31cebcba62e2b399bae5d6e79851aa82ec6686acedcf6f65"
+)
+depends=(
+    'libtiff'
+    'libpng'
+    'libjpeg'
+)
+useconfigure='true'
+
+pre_configure() {
+    run ./autogen.sh
```

Comment:
Just to make sure, there are no official preconfigured tarballs of this, right?

Reply:
Looks like [this](https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-1.3.1.tar.gz) is a thing, and it seems to be populated already.